docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,321 @@
+---
+title: "Add a TensorFlow Lite Modular Service"
+linkTitle: "Add a TensorFlow Lite Modular Service"
+weight: 70
+type: "docs"
+description: "Add a MLModel modular-resource-based service which uses TensorFlow Lite to classify audio samples."
+tags: ["ml", "model training", "services"]
+# SMEs: Andrew Morrow
+---
+
+Viam provides an example [modular resource](extend/modular-resources) written in C++ that extends the [ML model](/services/ml/) service to run any TensorFlow Lite model.
+
+The example files can be found in the [Viam C++ SDK](https://github.com/viamrobotics/viam-cpp-sdk):
+
+- [`example_mlmodelservice_tflite.cpp`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/src/viam/examples/modules/example_mlmodelservice_tflite.cpp) - an example module which provides a `MLModelService` modular resource capable of running any TensorFlow Lite model.
+- [`example_audio_classification_client.cpp`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/src/viam/examples/mlmodel/example_audio_classification_client.cpp) - an example inference client which generates audio samples and invokes the `example_mlmodelservice_tflite` module to classify those samples using the `yamnet/classification` TensorFlow Lite model.
+
+This tutorial walks you through everything necessary to start using these example files with your robot, including building the C++ SDK, configuring your robot and installing `viam-server`, and generating results with the example inference client program.
+
+## Build the C++ SDK
```

Comment:
Correct, the build steps build both the SDK and its included examples (including this example module + inference client).

I find value in replicating instructions here. Currently the user must read and comprehend [this](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/README.md) and [this](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md). Expert users may not need this hand holding, but all the rest of us -- for example: me -- probably benefit from having all the gotchas and caveats and probably-affected-bys in one place, imo. I struggled to get a reliable build going in the current alpha state of the SDK, and I feel that users are likely to experience similar.

Perhaps with beta or rc, the in-repo steps can be enhanced (happy to help!) to standardize and land on ease-of-use wins (like making VIAMCPPSDK_USE_DYNAMIC_PROTOS` the default!), but until then I vote to replicate.

Reply:
Should this then move to program/sdk and then just be linked from here ?

Reply:
Here: https://docs.viam.com/program/apis/ ? Hmm, that seems like more of a avail endpoints reference, rather than a build guide. No other SDK building content there either.

We could create a new section that handles this for all SDKs, but then that would definitely dupe other SDK repo content, as with this one.

I'd just advise waiting for cpp SDK to move out of alpha, to start the convo with cpp team about updating their repo's docs, and to then move this section out to README and BUILDING.

I just think we're too much in-between to standardize either side.

Reply:
No I was thinking here- and then making the install tab very long for c++  https://docs.viam.com/program/

Reply:
Gotcha. I see it now, it could definitely live there.

Would you consider it for a separate PR? I think I'd want to involve the CPP team if we're going to try to make a one-size-fits-all (appropriate for /program landing page) build guide. 

The build steps as presented in this tutorial are not one-size-fits all, and would need to be considered with expanded considerations if so, which would be a non-trivial endeavor.